providers/fireflies/README.md

```diff
@@ -4,10 +4,14 @@
 ## Supported Objects 
 Below is an exhaustive list of objects & methods supported on the objects
 
------------------------------------
-| Object     | Resource  | Method |
-| -----------| ----------| -------|
-| Users      | user      | read   |
-| Transcripts| transcript| read   |
-| Bites      | bite      | read   |
------------------------------------
\ No newline at end of file
+--------------------------------------------------------------------------
+| Object             | Resource                        | Method          |
+| -------------------| --------------------------------| ----------------|
+| Users              | user                            | read            |
+| Transcripts        | transcript                      | read            |
+| Bites              | bite (read), createBite (write) | read and write  |
+| AddToLiveMeeting   | addToLiveMeeting                | write           |
```

Comment:
our objects need to be unchanged **except** for when they aren't resources or entities (think yes for noun, no for verbs) - so this should ideally be a `liveMeeting` object which we can read/write to, and we should run the appropiate query for that